tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,31 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        upstream_name = u'busybox'
```

Comment:
Following previous style. Probably, was implemented in that way for better tests readability